cdap-common/src/main/java/co/cask/cdap/common/HttpExceptionHandler.java

```diff
@@ -52,4 +59,9 @@ public void handle(Throwable t, HttpRequest request, HttpResponder responder) {
       responder.sendStatus(HttpResponseStatus.INTERNAL_SERVER_ERROR);
     }
   }
+
+  private void logWarnings(HttpRequest request, Throwable t) {
+    LOG.trace("Error in handling request={} {} for user={}:", request.getMethod().getName(), request.getUri(),
```

Comment:
The method suggests that it is logging WARNING... but in fact it's logging trace. So, better rename the method.